### PR TITLE
fix menu button not clickable on android

### PR DESCRIPTION
**Pull request by kamiranoff**
#220

### CODE REVIEW

File: src/screens/DeckDetail/components/Menu.tsx [hunk 1]
```diff
@@ -136,35 +140,41 @@ const Menu: FC<Props> = ({ onShufflePress, onSortPress, onSharePress }) => {
...
-        text="Sort cards by incorrect first"
-        labelAnimatedStyle={labelStyle}
-        viewAnimatedStyle={sortStyle}
-        onPress={handleSortPress}
-      />
+    <View>
+      <View
+        style={[
+          styles.container,
+          isOpen ? { height: MAX_TRANSITION + ICON_SIZE, width: WINDOW_WIDTH } : null,
```
Comment (kamiranoff) [inline src/screens/DeckDetail/components/Menu.tsx:147]: That was basically the fix. A bit strange. But you can only click on the View. And with the animation and translates, the buttons where outside of this view. The fix is to make the vue big enough when the menu is open

File: src/screens/DeckDetail/components/TransitionedCards.tsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ const styles = StyleSheet.create({
   },
   container: {
     flexDirection: 'row',
-    paddingBottom: 400, // TODO: dynamic padding bottom
+    paddingBottom: 100,
```
Comment (kamiranoff) [inline src/screens/DeckDetail/components/TransitionedCards.tsx:92]: Now the scrollview takes the whole screen. Just added some padding to see the bottom of the last elements on the list

File: src/screens/DeckDetail/index.tsx [hunk 3]
```diff
@@ -125,7 +125,7 @@ const DeckDetail: FC<Props> = ({
           goodAnswersTotal={goodAnswers}
         />
       </View>
-      <AnimatedView top={500} scaleRatio={0.9} duration={500}>
+      <AnimatedView top={500} scaleRatio={0.9} duration={500} styles={{ flex: 1 }}>
```
Comment (kamiranoff) [inline src/screens/DeckDetail/index.tsx:128]: inline style!

File: src/screens/DeckDetail/components/TransitionedCards.tsx [hunk 4]
```diff
@@ -50,7 +50,7 @@ const TransitionedCards = forwardRef<TransitioningView, Props>(
 
       return (
         <View
-          key={item.id}
+          key={`cardlist-${item.id || item.frontendId}`}
```
Comment (kamiranoff) [inline src/screens/DeckDetail/components/TransitionedCards.tsx:53]: if offline, item.id is null.

File: src/screens/DeckDetail/components/Menu.tsx [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
 import { theme } from '../../../utils';
+import { isLargeDevice, WINDOW_WIDTH } from '../../../utils/device';
 
 interface Props {
   onShufflePress: () => void;
   onSortPress: () => void;
   onSharePress: () => void;
 }
 
+const MAX_TRANSITION = 210;
```
Comment (kamiranoff) [inline src/screens/DeckDetail/components/Menu.tsx:14]: here part of the fix

**Review by heron2014 — APPROVED** [review]

**Pull request merged**